content/en/guides/starting.md

```diff
@@ -29,7 +29,7 @@ Open source projects provide freedom to collaborate – even among competitors i
 
 ## When to create an open source project
 
-The decision to release or create a new open source project depends on your circumstances. Your company should first achieve a certain level of open source mastery by using open source software and contributing to existing projects. This is because consuming can teach you how to leverage external projects and developers to build your products. And participation can bring more fluency in the conventions and culture of open source communities. (See our guides on [Using Open Source Code](https://todogroup.org/guides/using-open-source/) and [Participating in Open Source Communities](https://todogroup.org/guides/participating/)) But once you have achieved open source fluency, the best time to start launching your own open source projects is simply “early” and “often.”
+The decision to release or create a new open source project depends on your circumstances. Your company should first achieve a certain level of open source mastery by using open source software and contributing to existing projects. This is because consuming can teach you how to leverage external projects and developers to build your products. And participation can bring more fluency in the conventions and culture of open source communities. (See our guides on [Using Open Source Code](https://todogroup.org/resources/guides/using-open-source-code/) and [Participating in Open Source Communities](https://todogroup.org/resources/guides/participating-in-open-source-communities/)) But once you have achieved open source fluency, the best time to start launching your own open source projects is simply “early” and “often.”
```

Comment:
Good work.  
We'd better split this PR , one for fixing the URL issues in English version to sync these updates to our guys in English , and another for Chinese version translation .

Reply:
done